Formula/rmlint.rb

```diff
@@ -21,7 +21,18 @@ class Rmlint < Formula
   depends_on "json-glib"
   depends_on "libelf"
```

Comment:
Haven't found a way to get `libelf` to be detected by Scons (both Linux and macOS).
I tried adding to CFLAGS but it didn't work inside Formula.
When I tried in a local copy, exporting CFLAGS worked, so not sure why.